src/ui/projection_switch.js

```diff
@@ -0,0 +1,43 @@
+module.exports = function(context) {
+
+  return function(selection) {
+  
+    var projections = [
+      {
+        label: 'Globe',
+        value: 'globe'
+      },
+      {
+        label: 'Mercator',
+        value: 'mercator'
+      }
+    ];
```

Comment:
Fabulous addition, FYI. I particularly love the behavior of maintaining the viewport centroid and approximate zoom.

Reply:
🤓